docs/ext/mpd.rst

```diff
@@ -64,6 +64,7 @@ See :ref:`config` for general help on configuring Mopidy.
 .. confval:: mpd/hostname
 
     Which address the MPD server should bind to.
+    This can be a network address or a file path.
```

Comment:
Sure, will do! To save another possible commit loop, how does this sound?  
_This can be a network address or the path to a Unix socket. Paths are expanded as in the Core configuration._